test/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/NoBusinessPresentControllerSpec.scala

```diff
@@ -48,17 +48,17 @@ class NoBusinessPresentControllerSpec
   "NoBusinessPresentControllerSpec" when {
 
     "handling request to get No Business Present" must {
-      def performAction() = controller.getNoBusinessPresent(FakeRequest())
+      def performAction = controller.getNoBusinessPresent(FakeRequest())
```

Comment:
def performAction() should be 

```
def performAction
```

As it has a non-unit value and the () was being relied on elsewhere to make it passed as an anonymous function. This was raising a warning